pkgs/applications/science/biology/macse/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "macse";
+  version = "2.03";
+
+  src = fetchurl {
+    url = "https://bioweb.supagro.inra.fr/${pname}/releases/${pname}_v${version}.jar";
+    sha256 = "0jnjyz4f255glg37rawzdv4m6nfs7wfwc5dny7afvx4dz2sv4ssh";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+  donUnpack = true; # not working?
```

Comment:
I don't see a reference to `dontUnpack` in the `nixpkgs` manual.

Reply:
Well, it's not documented yet I think, but present in many packages and only working in the unstable branch (see #65434). Furthermore, I did a typo! Well I remove this line... Thanks!